recipe/0002-upstream-patch-for-pypy.patch

```diff
@@ -0,0 +1,75 @@
+From ccdb775abe50125a845362fd0b65f1189c02e78b Mon Sep 17 00:00:00 2001
+From: Giampaolo Rodola <g.rodola@gmail.com>
+Date: Thu, 9 Nov 2023 14:34:11 -0800
+Subject: [PATCH] fix #2325: fix compilation on PyPy
+
+---
+ HISTORY.rst             |  5 +++++
+ psutil/_psutil_common.c | 12 ++++++++++++
+ psutil/_psutil_common.h | 14 +++++++++-----
+ 3 files changed, 26 insertions(+), 5 deletions(-)
+
+diff --git a/HISTORY.rst b/HISTORY.rst
+index 6044d8c50..0c62d823c 100644
+--- a/HISTORY.rst
++++ b/HISTORY.rst
+@@ -10,6 +10,11 @@ XXXX-XX-XX
+ - 2324_: enforce Ruff rule `raw-string-in-exception`, which helps providing
+   clearer tracebacks when exceptions are raised by psutil.
+ 
++**Bug fixes**
++
++- 2325_, [PyPy]: psutil did not compile on PyPy due to missing
++  `PyErr_SetExcFromWindowsErrWithFilenameObject` cPython API.
++
+ 5.9.6
+ =====
+ 
```

Comment:
Let's drop this part of the patch. Looks like it causes conflicts. Besides it is just updating the changelog, which don't need to worry about here